src/definitions.ts

```diff
@@ -26,6 +26,17 @@ declare module '@capacitor/cli' {
        * @example "Splashscreen"
        */
       imageName?: string;
+      /**
+       * Configure the content mode of displayed image. 
+       * Accepts values "center", "scaleToFill", "scaleAspectFit", "scaleAspectFill".
+       * Defaults to UIView.ContentMode.center in case of a different value or no value provided.
+       *
+       * Only available for iOS.
+       *
+       * @default ""
```

Comment:
Shouldn't that be "center"?

Reply:
Good catch, thank you! Will update asap

Reply:
I'd leave it blank as the value is not required, if not provided it automatically defaults to center. Just to leave it backward compatible as well.